tests/composite_pk/test_models.py

```diff
@@ -155,3 +155,9 @@ def test_permissions(self):
         self.assertEqual(4, token.permission_set.count())
         self.assertEqual(4, user.permission_set.count())
         self.assertEqual(4, comment.permission_set.count())
+
+    def test_refresh_foreign_object(self):
+        comment = Comment.objects.get(pk=self.comment_1.pk)
+        comment.user = None
+        comment.refresh_from_db()
+        self.assertEqual(comment.user, self.user_1)
```

Comment:
The query here seems redundant since we already have `self.comment_1`.

```suggestion
        self.comment_1.user = None
        self.comment_1.refresh_from_db()
        self.assertEqual(self.comment_1.user, self.user_1)
```
which is the same as Jacob's test.